app/lib/proxy_config_affecting_changes.rb

```diff
@@ -10,11 +10,13 @@ module ProxyRuleExtension
       after_commit :issue_proxy_affecting_change_events
 
       def issue_proxy_affecting_change_events
+        return unless owner
```

Comment:
There is no harm with this :+1: but in which case this is possible? :thinking:

Reply:
If it has been deleted but the ProxyRule was still pending to execute this callback.